### PR TITLE
Coordinates are null when there is no audio

### PR DESCRIPTION
**Pull request by raffi07**
changed active_source_id variable to active_source and added possibility to send null values if activity is below threshold

### CODE REVIEW

File: src/server/socket_server.py [hunk 1]
```diff
@@ -11,9 +11,8 @@
 remainingTrack = ''
 stream = ''
 string = ''
-active_source_id = ''
 
-def create_json(x,y,z):
+def create_json(x=None,y=None,z=None):
```
Comment (solodezaldivar) [inline src/server/socket_server.py]: test message

File: src/server/socket_server.py [hunk 2]
```diff
@@ -28,27 +27,33 @@ def create_json(x,y,z):
 
 def process(msg):
     data = ''
-    active_source_id = ''
     
     try:
         data = json.loads(msg)
         for source in data['src']:
-            if  active_source['id'] == '':
+            if not active_source:
```
Comment (solodezaldivar) [inline src/server/socket_server.py]: good choice for if statement

File: src/server/socket_server.py (hunk 1 above)
Comment (raffi07) [reply src/server/socket_server.py]: test answer

File: src/server/socket_server.py (hunk 2 above)
Comment (raffi07) [reply src/server/socket_server.py]: thanks pal

**Review by raffi07 — COMMENTED** [review]
:ambulance: Fixed threshold, added tests

**Pull request closed without merging**